--- inn-2.4.1/storage/cnfs/cnfs.h.old	2004-12-15 17:38:22.038029888 +0100
+++ inn-2.4.1/storage/cnfs/cnfs.h	2004-12-15 17:38:22.037030040 +0100
@@ -17,4 +17,12 @@
 void cnfs_printfiles(FILE *file, TOKEN token, char **xref, int ngroups);
 void cnfs_shutdown(void);
 
+/* Patch for POSIX
+*/
+#ifndef MADV_SEQUENTIAL 
+  #ifdef  POSIX_MADV_SEQUENTIAL
+    #define MADV_SEQUENTIAL  POSIX_MADV_SEQUENTIAL
+  #endif
+#endif
+
 #endif
