diff -up inn-2.5.2/frontends/Makefile.smp inn-2.5.2/frontends/Makefile
--- inn-2.5.2/frontends/Makefile.smp	2010-08-11 20:54:46.268104983 +0200
+++ inn-2.5.2/frontends/Makefile	2010-08-11 20:58:04.429916114 +0200
@@ -85,7 +85,7 @@ ovdb_init:	ovdb_init.o    $(BOTH)   ; $(
 ovdb_monitor:	ovdb_monitor.o $(BOTH)   ; $(LINK) ovdb_monitor.o $(STORELIBS)
 ovdb_server:	ovdb_server.o  $(BOTH)   ; $(LINK) ovdb_server.o  $(STORELIBS)
 ovdb_stat:	ovdb_stat.o    $(BOTH)   ; $(LINK) ovdb_stat.o    $(STORELIBS)
-rnews:		rnews.o        $(LIBINN) ; $(LINK) rnews.o        -pie $(STORELIBS)
+rnews:		rnews.o        $(BOTH)   ; $(LINK) rnews.o        -pie $(STORELIBS)
 sm:		sm.o           $(BOTH)   ; $(LINK) sm.o           $(STORELIBS)
 
 ovdb_init.o: ovdb_init.c
diff -up inn-2.5.2/innfeed/Makefile.smp inn-2.5.2/innfeed/Makefile
--- inn-2.5.2/innfeed/Makefile.smp	2010-08-11 20:47:15.883915244 +0200
+++ inn-2.5.2/innfeed/Makefile	2010-08-11 20:47:15.875915561 +0200
@@ -54,9 +54,13 @@ $(FIXSCRIPT):
 
 INNFEEDLIBS	= $(LIBSTORAGE) $(LIBHIST) $(LIBINN) $(STORAGE_LIBS) $(LIBS)
 
-config_y.c config_y.h: configfile.y
+y.tab.c y.tab.h: configfile.y
 	$(YACC) -d $?
+
+config_y.h: y.tab.h
 	mv y.tab.h config_y.h
+
+config_y.c: y.tab.c
 	mv y.tab.c config_y.c
 
 config_l.c: configfile.l
