Index: doc/pod/actsync.pod
===================================================================
--- a/doc/pod/actsync.pod	(revision 9473)
+++ b/doc/pod/actsync.pod	(revision 9474)
@@ -524,24 +524,24 @@
 
 =over 2
 
-=item 0
+=item 0Z<>
 
 No debug or status reports (default).
 
-=item 1
+=item 1Z<>
 
 Print summary, but only if work was needed or done.
 
-=item 2
+=item 2Z<>
 
 Print actions, exec output, and summary, but only if work was needed or
 done.
 
-=item 3
+=item 3Z<>
 
 Print actions, exec output, and summary.
 
-=item 4
+=item 4Z<>
 
 Full debug output.
 
