diff -up inn-2.5.1/frontends/Makefile.pie inn-2.5.1/frontends/Makefile
--- inn-2.5.1/frontends/Makefile.pie	2009-10-12 20:24:04.000000000 +0200
+++ inn-2.5.1/frontends/Makefile	2009-10-13 21:51:38.000000000 +0200
@@ -85,7 +85,7 @@ ovdb_init:	ovdb_init.o    $(BOTH)   ; $(
 ovdb_monitor:	ovdb_monitor.o $(BOTH)   ; $(LINK) ovdb_monitor.o $(STORELIBS)
 ovdb_server:	ovdb_server.o  $(BOTH)   ; $(LINK) ovdb_server.o  $(STORELIBS)
 ovdb_stat:	ovdb_stat.o    $(BOTH)   ; $(LINK) ovdb_stat.o    $(STORELIBS)
-rnews:		rnews.o        $(LIBINN) ; $(LINK) rnews.o        $(STORELIBS)
+rnews:		rnews.o        $(LIBINN) ; $(LINK) rnews.o        -pie $(STORELIBS)
 sm:		sm.o           $(BOTH)   ; $(LINK) sm.o           $(STORELIBS)
 
 ovdb_init.o: ovdb_init.c
diff -up inn-2.5.1/innd/Makefile.pie inn-2.5.1/innd/Makefile
--- inn-2.5.1/innd/Makefile.pie	2009-10-12 20:24:04.000000000 +0200
+++ inn-2.5.1/innd/Makefile	2009-10-13 21:53:11.000000000 +0200
@@ -44,7 +44,7 @@ perl.o:		perl.c   ; $(CC) $(CFLAGS) $(PE
 python.o:	python.c ; $(CC) $(CFLAGS) $(PYTHON_CPPFLAGS) -c python.c
 
 innd: $(OBJECTS) $(LIBSTORAGE) $(LIBHIST) $(LIBINN)
-	$(LIBLD) $(LDFLAGS) -o $@ $(OBJECTS) $(INNDLIBS)
+	$(LIBLD) -pie $(LDFLAGS) -o $@ $(OBJECTS) $(INNDLIBS)
 
 tinyleaf: tinyleaf.o $(LIBINN)
 	$(LIBLD) $(LDFLAGS) -o $@ tinyleaf.o $(LIBINN) $(LIBS)
