diff -up inn-2.5.1/storage/buffindexed/ovmethod.mk.shared inn-2.5.1/storage/buffindexed/ovmethod.mk
--- inn-2.5.1/storage/buffindexed/ovmethod.mk.shared	2009-12-01 16:50:37.174811272 +0100
+++ inn-2.5.1/storage/buffindexed/ovmethod.mk	2009-12-01 16:53:24.914809560 +0100
@@ -1,8 +1,8 @@
 # This rule requires a compiler that supports -o with -c.  Since it's normally
 # used by developers, that should be acceptable.
-buffindexed/buffindexed_d.o: buffindexed/buffindexed.c
-	$(CC) $(CFLAGS) -DBUFF_DEBUG -c -o $@ buffindexed/buffindexed.c
+buffindexed/buffindexed_d.$(EXTOBJ): buffindexed/buffindexed.c
+	$(LIBCC) $(CFLAGS) -DBUFF_DEBUG -c -o $@ buffindexed/buffindexed.c
 
-buffindexed/buffindexed_d: buffindexed/buffindexed_d.o libstorage.$(EXTLIB) $(LIBHIST)
-	$(LIBLD) $(LDFLAGS) -o $@ buffindexed/buffindexed_d.o \
+buffindexed/buffindexed_d: buffindexed/buffindexed_d.$(EXTOBJ) libstorage.$(EXTLIB) $(LIBHIST)
+	$(LIBLD) $(LDFLAGS) -o $@ buffindexed/buffindexed_d.$(EXTOBJ) \
 	    $(LIBSTORAGE) $(LIBHIST) $(LIBINN) $(STORAGE_LIBS) $(LIBS)
diff -up inn-2.5.1/storage/tradindexed/ovmethod.mk.shared inn-2.5.1/storage/tradindexed/ovmethod.mk
--- inn-2.5.1/storage/tradindexed/ovmethod.mk.shared	2009-12-01 16:53:35.644811390 +0100
+++ inn-2.5.1/storage/tradindexed/ovmethod.mk	2009-12-01 16:53:59.964810001 +0100
@@ -1,6 +1,6 @@
-tradindexed/tdx-util.o: tradindexed/tdx-util.c
-	$(CC) $(CFLAGS) -c -o $@ tradindexed/tdx-util.c
+tradindexed/tdx-util.$(EXTOBJ): tradindexed/tdx-util.c
+	$(LIBCC) $(CFLAGS) -c -o $@ tradindexed/tdx-util.c
 
-tradindexed/tdx-util: tradindexed/tdx-util.o libstorage.$(EXTLIB) $(LIBHIST)
-	$(LIBLD) $(LDFLAGS) -o $@ tradindexed/tdx-util.o \
+tradindexed/tdx-util: tradindexed/tdx-util.$(EXTOBJ) libstorage.$(EXTLIB) $(LIBHIST)
+	$(LIBLD) $(LDFLAGS) -o $@ tradindexed/tdx-util.$(EXTOBJ) \
 	    $(LIBSTORAGE) $(LIBHIST) $(LIBINN) $(STORAGE_LIBS) $(LIBS)
